app/src/main/java/com/sun/livescore/data/TeamDataSource.kt

```diff
@@ -0,0 +1,11 @@
+package com.sun.livescore.data
+
+import com.sun.livescore.data.model.team.Team
+import io.reactivex.Single
+
+class TeamDataSource {
+    interface Local {
+        fun saveFaveTeams(team: Team)
```

Comment:
```suggestion
        fun saveFavTeam(team: Team)
```
Lưu có 1 team thôi mà để số nhiều là như nào =))